<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_toFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>To File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_inFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>In Files</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_toFile">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>611</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="edit_inFiles">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>611</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select name</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listwidget_name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>611</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_run">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_schedule">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>611</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_schedule">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>schedule</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb_selectall">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>select all</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressbar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>620</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_schedulekey">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>590</y>
      <width>611</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>570</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Secure key for apis.data.go (getHoliDeInfo)</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_quarter">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>71</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Quarter</string>
    </property>
    <widget class="QRadioButton" name="rbt_1q">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1Q</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbt_2q">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>2Q</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbt_3q">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>3Q</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbt_4q">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>4Q</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>620</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>status</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
